### PR TITLE
Hw1

### PR DESCRIPTION
**Pull request by halltim91**
Contains both exercises for homework 1.

Changes made:
* Added answers for css dinner game
* Added bootstrap formatting and functionality to palindrome app

### CODE REVIEW

File: hw1/02-palindrome.js [hunk 1]
```diff
@@ -1,3 +1,34 @@
 const elem = document.querySelector('input');
+const output = document.querySelector('#status')
 
 elem.addEventListener('input', handleInput);
+function handleInput(event) {
+    const input = event.target.value;
+    
+    if(input.length === 0){
+        output.textContent = " ";
+    } else if(isNaN(input)){ //doesn't really do anything since you can't enter letters into number input
```
Comment (NathanMoes) [inline hw1/02-palindrome.js:10]: Agreed, I had the same exp. Don't know why it is in the hw1 spec but oh well. Only thing is, it does allow e. As in a single e. So, idk

File: hw1/02-palindrome.js [hunk 2]
```diff
@@ -1,3 +1,34 @@
...
+            if(input[i] !== input[input.length - (1 + i)]){
+                setOutputText('No. Try again.', false);
+                return;
+            }
+        }
+        setOutputText('Yes. This is a palindrome!', true);
+    }
+}
+
+function setOutputText(text, isValid){
```
Comment (NathanMoes) [inline hw1/02-palindrome.js:25]: Like the modular approach here for styling!

**Review by NathanMoes — CHANGES_REQUESTED** [review]
Form is not quite responsive in the way that would look best. I would sugest using flex box aka d-flex etc. Form is too cramped. Other than that it works fine

**Comment by NathanMoes** [comment]
> Form is not quite responsive in the way that would look best. I would sugest using flex box aka d-flex etc. Form is too cramped. Other than that it works fine

It also it much smaller than in the hw1 spec

**Review by shrutivd — APPROVED** [review]
1. The use of d-flex box would be ideal to make the form responsive. 
2. "getElementByid" is faster and efficient than "querySelector" if we need to search for an element with the specified ID.

**Pull request merged**